<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CustomPage</class>
 <widget class="QWidget" name="CustomPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>2000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:white</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>351</width>
     <height>44</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Created</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Last seen</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="created_time_title">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>08.04.2022</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="last_visited_time_title">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>08.04.2022</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="start_testing_button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>150</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 16px 24px;

position: absolute;
width: 134px;
height: 51px;
left: 724px;
top: 315px;

background: #005DF7;
border: 2px solid rgba(255, 255, 255, 0.64);
border-radius: 12px;</string>
   </property>
   <property name="text">
    <string>Start testing</string>
   </property>
  </widget>
  <widget class="QPushButton" name="recom_notes_button">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>170</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Recom  to this note</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_button">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>170</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Save!</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_field">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>411</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add_question_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>141</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Add question!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="page_title">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>381</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">position: absolute;
width: 395px;
height: 77px;
left: 582px;
top: 120px;

font-family: 'Inter';
font-style: normal;
font-weight: 500;
font-size: 32px;
line-height: 77px;

color: #000000;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
